devito/ops/compiler.py

```diff
@@ -48,8 +40,11 @@ def jit_compile(self, soname, ccode, hcode):
                 self.code = f.read()
         except FileNotFoundError:
             warning("Couldn't find file: %s" % self.ops_src)
-        else:
-            self._compile_cuda(soname)
+
+        if configuration['platform'] == 'nvidiaX':
```

Comment:
really? shouldn't it be
```
if configuration['platform'] is NVIDIAX
```

like [here](https://github.com/opesci/devito/blob/e54d1c83b9d69ca406c5e9e718349b5d4066bd1b/devito/compiler.py#L370)
(note that the `==` there should and could actually be `is` -- exercise for you, understand why. Not that the `==` comparison is wrong, but the `is` comparison is conceptually better)

Reply:
thank you, fixed